djconnectwise/models.py

```diff
@@ -1488,8 +1508,6 @@ class Ticket(UpdateConnectWiseMixin, TimeStampedModel):
         'phase': 'phase'
     })
     SERVICE_EDITABLE_FIELDS.update({
```

Comment:
Can this be removed if it is going to be empty?

Reply:
> I don't understand the reference to source here though

Err, right, not sure what I was thinking at the time...